settings/PandoraSettings_Master_DUNEFD.xml

```diff
@@ -37,22 +42,23 @@
         <RecreatedClusterListName>RecreatedClusters</RecreatedClusterListName>
         <RecreatedVertexListName>RecreatedVertices</RecreatedVertexListName>
         <VisualizeOverallRecoStatus>false</VisualizeOverallRecoStatus>
+		<InputMCParticleListName>Input</InputMCParticleListName>
+        <PassMCParticlesToWorkerInstances>true</PassMCParticlesToWorkerInstances>
     </algorithm>
-
     <algorithm type = "LArEventValidation">
         <CaloHitListName>CaloHitList2D</CaloHitListName>
         <MCParticleListName>Input</MCParticleListName>
         <PfoListName>RecreatedPfos</PfoListName>
         <UseTrueNeutrinosOnly>false</UseTrueNeutrinosOnly>
         <PrintAllToScreen>false</PrintAllToScreen>
-        <PrintMatchingToScreen>true</PrintMatchingToScreen>
-        <WriteToTree>false</WriteToTree>
-        <OutputTree>Validation</OutputTree>
-        <OutputFile>Validation.root</OutputFile>
+        <PrintMatchingToScreen>false</PrintMatchingToScreen>
+        <WriteToTree>false</WriteToTree> <!-- change -->
+        <OutputTree>test_mousam</OutputTree>
+        <OutputFile>test_mousam.root</OutputFile>
```

Comment:
Names here should probably revert to the standard Validation and Validation.root versions.